chatbot/app.py

```
@@ -87,4 +87,4 @@ def chat():
         return jsonify({"error": str(e)})
 
 if __name__ == '__main__':
-    app.run(host="0.0.0.0", port=5000,debug=True)
+    app.run(host="0.0.0.0", port=5000,debug=True)
```

Ruff (0.14.10)

Possible binding to all interfaces

(S104)